[MRG] Sample weights for ElasticNetCV (#16449)
scikit-learn/scikit-learn: doc/whats_new/v1.0.rst

  complexity in the number of samples.
  :pr:`10027` by :user:`Albert Thomas <albertcthomas>`.

- |Feature| Add `sample_weight` parameter to
  :class:`linear_model.LassoCV` and :class:`linear_model.ElasticNetCV`.

>   This is accompanied by the new parameter `cv_weighted` that allows

Reviewer: ```suggestion
  This is accompanied by the new parameter `use_weights_in_cv` that allows
```

>   fine-control whether or not to use the sample weights for averaging
>   scores over cv-folds.

Reviewer: I think this reads better:
```suggestion
  fine-control on the use of sample weights for averaging scores over
  cv-folds.
```
